modules/tile-converter/src/converter-cli.ts

```diff
@@ -177,6 +180,7 @@ function printHelp(): void {
     '--tileset [tileset.json file (3DTiles) / http://..../SceneServer/layers/0 resource (I3S)]'
   );
   console.log('--input-type [tileset input type: I3S or 3DTILES]');
+  console.log('--tiles-version [3dtile version: 1.0 or 1.1, default: 1.1]');
```

Comment:
```suggestion
  console.log('--output-version [3dtile version: 1.0 or 1.1, default: 1.1]');
```

Reply:
Add to documentation something like "This option supports only `1.0`/`1.1` values for 3DTiles output. I3S output version setting is not supported yet

Reply:
Done.